working/augmentation-libraries/feature-specification.md

```diff
@@ -785,45 +789,47 @@ some_lib.dart       | some_augment.dart
 '-----------------' | '-----------------'
 ```
 
-Each library has its own namespace chain from its own member bodies out to its
-own import namespace. But in the middle, each passes through the shared class
-namespaces for the instance and static members and the shared top level
-declaration scope.
+Each library augmentation has its own namespace chain from its own member bodies
+out to its own import namespace. But in the middle, each passes through the
+shared class namespaces for the instance and static members and the shared top
+level declaration scope.
 
 This implies that the bodies of members cannot be resolved until after
 augmentations have been merged. We don't know what namespace `isEven()` or
 `_isOdd()` will be found in until the namespace for the class has had *all*
 augmentations merged in.
 
 Fortunately, augmentations can be applied purely syntactically. In order to
-merge augmentations into the main library, we only need to match declarations
-by name. So a Dart compiler can theoretically:
+merge augmentations into the augmented library, we only need to match
+declarations by name. So a Dart compiler can theoretically:
 
-1.  Parse the main library and all of its augmentations.
+1.  Parse the augmented library and all of its augmentations.
 2.  Merge the augmentations to determine the complete set of declarations in
     all types.
-3.  Resolve and type-check the main library and all of its augmentations now
-    that all type namespaces are complete.
+3.  Resolve and type-check the augmented library and all of its augmentations
+    now that all type namespaces are complete.
 
 ## Syntax
 
-The grammar changes are fairly simple. A main library can apply an augmentation
-using a new directive:
+The grammar changes are fairly simple. A library can apply an augmentation using
```

Comment:
Heh, I suppose this first sentence as a little too optimistic about simplicity. :)